CONTRIBUTING.md

```diff
@@ -364,6 +364,13 @@ To cut a release:
 
 * Create a release branch from `main` (`git checkout -b release-v0.0.1`)
 * Remove the `-unreleased` from the `version` field in `Cargo.toml` and `flake.nix`
+* Update the flake.lock and Cargo.lock, and Rust dependencies are up-to-date with the following:
```

Comment:
```suggestion
* Ensure the flake.lock and Cargo.lock, and Rust dependencies are up-to-date with the following:
```

Reply:
In theory these updates would happen prior to the release PR, mind making this the first point?